spec/components/calendar_spec.rb

```diff
@@ -0,0 +1,24 @@
+require("rails_helper")
+describe("CalendarComponent", type: :view) do
```

Comment:
I think we should decide at this point what format the view tests will take. i.e. should they be [feature tests](https://github.com/rspec/rspec-rails/blob/main/features/view_specs/view_spec.feature)?

Reply:
I've deliberately put aside the question of changing test types (see above), but I think view specs are appropriate for the components, at the very least (https://rspec.info/features/6-0/rspec-rails/view-specs/view-spec/) - they're for testing templates independently of a controller, so it seems fine for component tests (where we want to test that the component is being rendered but could be in multiple controllers). Existing specs for components  (outside of the gem, which uses minitest) seem to use view specs (https://github.com/alphagov/collections/blob/main/spec/components/document_navigation_list_spec.rb)